apps/condo/migrations/20230511104910-0270_auto_20230511_0549.js

```diff
@@ -0,0 +1,28 @@
+// auto generated by kmigrator
+// KMIGRATOR:0270_auto_20230511_0549:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMSBvbiAyMDIzLTA1LTExIDA1OjQ5Cgpmcm9tIGRqYW5nby5kYiBpbXBvcnQgbWlncmF0aW9ucwoKCmNsYXNzIE1pZ3JhdGlvbihtaWdyYXRpb25zLk1pZ3JhdGlvbik6CgogICAgZGVwZW5kZW5jaWVzID0gWwogICAgICAgICgnX2RqYW5nb19zY2hlbWEnLCAnMDI2OV9yZW1vdmVfYmFua3RyYW5zYWN0aW9uX2JhbmtfdHJhbnNhY3Rpb25fdW5pcXVlX251bWJlcl9kYXRlX29yZ2FuaXphdGlvbl9hbmRfbW9yZScpLAogICAgXQoKICAgIG9wZXJhdGlvbnMgPSBbCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+        BEGIN;
+            --
+            -- [CUSTOM] Set Statement Timeout to some large amount - 25 min (25 * 60 => 1500 sec)
+            --
+            SET statement_timeout = '1500s';  
+            
+            UPDATE "BankTransaction" b
+            SET "importId" = CONCAT (CAST(b."date" as varchar), CAST('_' as varchar), CAST(b."number" as varchar))
```

Comment:
It works without explicit type casting

```suggestion
            SET "importId" = CONCAT (b."date", '_', b."number")
```